docs/source/algorithms/SetSample-v1.rst

```diff
@@ -63,14 +68,20 @@ procedure is used when trying to find a named definition, e.g ``CRYO-01``:
 
   - repeat for the facility directories if not found in for the specific instrument
 
+Container Geometry and Material
```

Comment:
The documentation for the material\geometry inputs seems to be combined for sample and container. But this sits below a header called "Container Geometry and Material". Maybe you need to have a section for "Geometry" and "Material" and then refer back to these from the "Container Geometry and Material" section? It's quite nice if there's a header for each of the input parameters